client/src/components/Header.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from "react"
+import styled from "styled-components"
+
+const HeaderContainer = styled.div`
+    align-items: center;
+    background: #eee;
+    color: black;
+    display: flex;
+    height: 15vh;
+    justify-content: center;
+    width: 100%;
+`
+
+const Logo = styled.div`
+    font-size: 1.75rem;
+`
+
+export default class Header extends Component {
+    render() {
+        return (
+            <HeaderContainer>
+                <Logo><em>Quick Quack</em></Logo>
```

Comment:
I think it's unnecessary to have `em` inside the `Logo`, just include the `em` styling into the `Logo` component itself.